packages/core/src/RPA/core/logger.py

```diff
@@ -11,6 +12,14 @@
 from RPA.core.helpers import required_param
 
 
+logging.captureWarnings(True)
```

Comment:
I wonder if this could have some unforeseen side-effects when used within RF. It seems like RF itself is using the warnings module internally, but doesn't call this

Reply:
Yes, definitely, good catch! Most probably would spam too much the dev.

I fixed it with a temporary switch to capturing logging only when using this utility. I've seen the idiom in some [examples](https://www.programcreek.com/python/example/107920/logging.captureWarnings).